docs/topics/mta-web-archetypes.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+// * topics/mta-assessment-module.adoc
+
+
+[id="mta-archetypes_{context}"]
+= Archetypes
+
+{ProductFullName} includes *archetypes* that groups applications that are assessed for migration, according to common characteristics. The purpose of archetypes is to help you perform bulk assessments and reviews of applications.
+
+Archetypes store information about the different application types in the organization based on their taxonomy, and automatically classify applications based on their associated tags. If the tags of an application match the criteria tags of an archetype, then the application is associated with the archetype.
+
+Archetypes enable {ProductShortName} you to apply questionnaires populated with questions that apply to common application characteristics.
+
+Creation of an archetype is defined by a series of *tags*, *stakeholders*, and *stakeholder groups*.
+
+[id="mta-define-archetype_{context}"]
+== Defining an archetype
+
+. Open the {ProductShortName} web console.
+. In the left menu, click *Archetypes* and then click *Create new archetype*.
+. In the form that opens, enter the following information for the new archetype:
+
+.. *Name* of the new archetype. (mandatory)
+.. *Description* of the new archetype. (optional)
+.. *Criteria Tags* are the tags that associate the assessed applications with the archetype. (mandatory). If criteria tags are updated, the process to calculate the applications, with which the archetype is associated with, is triggered again.
+.. *Archetype Tags* are the tags that the archetype assesses in the application. (mandatory)
+.. *Stakeholder(s)* are specific stakeholders involved in the application development and migration. (optional)
+.. *Stakeholders Group(s)* are defined groups of stakeholders. (optional)
+
+. Click *Create*. The new archetype appears in the list.
+
+After creating an archetype, applications in the inventory are automatically associated to an archetype if its associated tags match with the ones associated to the archetype.
+
```

Comment:
After you create an archetype, every application in the inventory whose tags exactly match those of the archetype is automatically associated to that archetype. [Not sure about the exact;  remove "exactly" if it's incorrect"